recipe/meta.yaml

```diff
@@ -36,10 +36,25 @@ requirements:
     - traitlets
 
 test:
+  requires:
+    - nose
+    - setuptools
+    - wheel
+    - nose
+    - coverage
+    - nodejs
+    - phantomjs
```

Comment:
From a jupyter/notebook perspective it is definitely valuable to run the test suite on as many platforms as possible to get fast feedback at release time.

From a conda-forge perspective, the only reason for not wanting to run the tests is to avoid hogging CI resources for other projects. There is definitely a balance there, 30 seconds for running the tests is sensible, 30 minutes isn't - and there is a grey line somewhere between those two magic numbers 😄 . I'm very happy to leave that to the feedstock maintainers to make a call on.